examples/tox21/train_tox21.py

```diff
@@ -81,7 +86,20 @@ def main():
     predictor_ = predictor.build_predictor(
         method, args.unit_num, args.conv_layers, class_num)
 
-    train_iter = I.SerialIterator(train, args.batchsize)
+    iterator_type = args.iterator_type
+    if iterator_type == 'serial':
+        train_iter = I.SerialIterator(train, args.batchsize)
+    elif iterator_type == 'balanced':
+        from chainer_chemistry.iterators.balanced_serial_iterator import BalancedSerialIterator  # NOQA
```

Comment:
Pep8 recommends to import modules at the top of files. So, I suggest to move it upward if unless it is unavoidable.

Reply:
One note is that when user install library from `pip` command, this import does not work.
Therefore, original option 'serial' is still available in this implementation but 'balanced' is not available from pip library version=0.1.0.

Should I write `try ... except ImportError: print('if you want to use BalancedSerialIterator, please install library from master branch')` or something?

Reply:
I understood. Then, I want to turn down this suggestion and want you to revert it because the current code show warning about `BalancedSerialIterator` even if users want to use ordinal iterators.

Reply:
As we discussed offline, we will update the minor version when the library code cannot run the example code in the master branch and it is costly or makes the example complicated if we want to support backward compatibility.